jenkinsapi/utils/crumb_requester.py

```diff
@@ -16,6 +16,26 @@ def __init__(self, *args, **kwargs):
         self._baseurl = kwargs['baseurl']
         self._last_crumb_data = None
 
+    def get_url(self, url, params=None, headers=None, allow_redirects=True,
+                stream=False):
+        if self._last_crumb_data:
+            # first try request with previous crumb if available
+            response = self._get_url_with_crumb(
+                self._last_crumb_data, url, params,
+                headers, allow_redirects, stream
+            )
+            # code 403 might indicate that the crumb is not valid anymore
+            if response.status_code != 403:
+                return response
+
+        # fetch new crumb (if server has crumbs enabled)
+        if self._last_crumb_data is not False:
```

Comment:
there is already check above for `self._last_crumb_data` being `True`. Why this check is here?